executor/mpp_gather.go

```diff
@@ -139,6 +139,7 @@ func (e *MPPGather) Next(ctx context.Context, chk *chunk.Chunk) error {
 
 // Close and release the used resources.
 func (e *MPPGather) Close() error {
+	e.mppReqs = nil
```

Comment:
No needs for a GC language

Reply:
open-next-close() are involved multiple times during apply, we should deleted some resouces in close().